### PR TITLE
chore: Update installation.mdx

### PR DESCRIPTION
**Pull request by tarasyarema**
Updated the React getting started to the correct installation commands and snippets.

## Describe your changes

Updated the React getting started guide with the correct commands.

[Slack thread](https://trylatitude.slack.com/archives/C06RP64CSCQ/p1717503480856129)

## Checklist before requesting a review
- [ ] I have added thorough tests
- [ ] I have updated the documentation if necessary
- [ ] I have added a human-readable description of the changes for the release notes
- [ ] I have included a recorded video capture of the feature manually tested

### CODE REVIEW

File: docs/react/getting-started/installation.mdx [hunk 1]
```diff
@@ -10,12 +10,18 @@ description: 'How to install the react package in your project'
     ```bash
     npm install @latitude-data/react
     ```
+
+    You might need to install `tailwindcss` before if your project does not have it installed:
+
+    ```bash
+    npm install tailwindcss
+    ```
```
Comment (andresgutgon) [inline docs/react/getting-started/installation.mdx:18]: Tailwind is not a requirement tho. latitude use tailwind under the hood but people using react package can use any other CSS solution

Comment (tarasyarema) [reply docs/react/getting-started/installation.mdx:18]: hmm it gave me an error when installing w pnpm, so it means that its just a warning?

Comment (andresgutgon) [reply docs/react/getting-started/installation.mdx:18]: Sorry maybe I didn't understand. How is tailwind related to the package manager?

Comment (andresgutgon) [reply docs/react/getting-started/installation.mdx:18]: This is super interesting and can help us to improve the docs btw 💪

Comment (tarasyarema) [reply docs/react/getting-started/installation.mdx:18]: This is what I get from the terminal

```bash
➜  fe git:(master) pnpm add @latitude-data/react
 WARN  deprecated react-use-gesture@9.1.3: This package is no longer maintained. Please use @use-gesture/react instead
Downloading registry.npmjs.org/echarts/5.5.0: 10.4 MB/10.4 MB, done
Packages: +57 -1
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++-
Progress: resolved 983, reused 888, downloaded 55, added 57, done

dependencies:
+ @latitude-data/react 0.9.0

 WARN  Issues with peer dependencies found
.
└─┬ @latitude-data/react 0.9.0
  ├─┬ tailwindcss-animate 1.0.7
  │ └── ✕ missing peer tailwindcss@">=3.0.0 || insiders"
  └─┬ @latitude-data/client 0.11.0
    └─┬ tailwind-variants 0.2.1
      └── ✕ missing peer tailwindcss@"*"
✕ Conflicting peer dependencies:
  tailwindcss

Done in 11.5s
```

Comment (andresgutgon) [reply docs/react/getting-started/installation.mdx:18]: And do you use tailwind in your project? 

If you already was using Tailwind makes sense. But for a project that doesn't use Tailwind it shouldn't require it. We don't have it in peerDependencies https://github.com/latitude-dev/latitude/blob/main/packages/client/react/package.json#L46-L49

Comment (tarasyarema) [reply docs/react/getting-started/installation.mdx]: Ok, make sense, updated

**Review by andresgutgon — APPROVED** [review]
I think you're our first contribution : )

**Pull request merged**